cdap-tms/src/main/java/co/cask/cdap/messaging/service/CoreMessagingService.java

```diff
@@ -218,6 +218,10 @@ public void rollback(TopicId topicId, RollbackDetail rollbackDetail) throws Topi
 
   @Override
   protected void startUp() throws Exception {
+    // Upgrade the TMS Message and Payload Tables
```

Comment:
Please add a comment here that we need to acquire a distributed lock before doing this when TMS supports running multiple instances.